Vagrantfile

```diff
@@ -0,0 +1,44 @@
+Vagrant.configure("2") do |config|
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+
+  sudo apt-get update
+
+  # Install pyenv prerequisites
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \ 
+    libedit-dev libsqllite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \ 
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+  # Install pyenv
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+    echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.profile
+
+  SHELL
+
+  # Reload the shell to pick up pyenv
+  config.vm.provision :reload
```

Comment:
It looks like this is using a plugin, we should document that on the readme so users know to install it before trying to run the app through vagrant, but having said that, I think the intention here is to reload the VM before the next step to pick up the latest changes to our profile. We can instead do that explicitly, by adding a `source ~/.profile` line to load in the latest environment settings, which will allow us to combine our two provision steps